src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -1701,4 +1710,10 @@ mod tests {
         test_ok_case_zero_arg(&mut ctx, ScalarFuncSig::AddTimeDurationNull, Datum::Null);
         test_ok_case_zero_arg(&mut ctx, ScalarFuncSig::SubTimeDurationNull, Datum::Null);
     }
+
+    #[test]
+    fn test_add_sub_time_string_null() {
```

Comment:
Would you like to implement `SubTimeStringNull`?

Reply:
A PR can only implement one function，So I only implement AddTimeStringNull,All test used test_add_sub as it's prefix,so i used the test_add_sub even though there is no sub function.